### ISSUE TITLE
Ground truth edge maps for training

### ISSUE THREAD

**Issue by zjbthomas** [dong03/MVSS-Net#13]
Dear authors,

Thanks for sharing the source code of MVSS-Net and it is really a great work!

I am actually wondering how the edge loss is calculated during training.
As CASIV v2 does not provide ground truth edge maps, did you extract the edge maps from the ground truth pixel maps on your own? If so, could you please share how you did it?

Thank you!

**Comment by CarnoZhao** [comment]
Same question here. Could you please provide the detail of loss function implementation?

**Comment by dong03** [comment]
We extract edge maps mainly via cv2.findContours,  and CASIAv2's is available at [here](https://github.com/dong03/MVSS-Net#training-sets).

The loss function of segmentation is DICE loss, and the loss function of classification is BCE loss.
